### ISSUE TITLE
Extension Error: this.handleRawError is not a function

### ISSUE THREAD

**Issue by gustavomeloweb** [GoogleChrome/lighthouse#1725]
**Lighthouse Version**: 1.5.1
**Chrome Version**: 58.0.3011.0
**URL**: https://www.cers.com.br/
**Error Message**: this.handleRawError is not a function
**Stack Trace**:
 ```TypeError: this.handleRawError is not a function
    at Object.chrome.debugger.sendCommand.result [as callback] (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:8769:33)
    at safeCallbackApply (extensions::sendRequest:21:15)
    at handleResponse (extensions::sendRequest:74:7)```

**Comment by wardpeet** [comment]
@gustavomeloweb can you try version 1.5.2?

**Comment by brendankenny** [comment]
fixed in #1697/1.5.2
